go.mod

```diff
@@ -11,11 +11,11 @@ require (
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/huandu/xstrings v1.3.0 // indirect
 	github.com/mitchellh/copystructure v1.0.0 // indirect
-	github.com/onsi/ginkgo v1.14.2
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/ginkgo v1.11.0
+	github.com/onsi/gomega v1.8.1
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.0.0
-	go.uber.org/zap v1.16.0
+	go.uber.org/zap v1.10.0
```

Comment:
looks like this is downgrading the dependabot changes, mind if you undo these ?

Reply:
I synced the go.mod and go.sum with master